help/en/html/apps/PanelPro/GettingStarted.shtml

```diff
@@ -1,19 +1,12 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
-"http://www.w3.org/TR/html4/strict.dtd">
+<!DOCTYPE html>
+
```

Comment:
JavaHelp really only supports HTML 4.01.  It doesn't care about the DOCTYPE (JavaHelp is really not very bright), but changing the doctype to HTML i.e. HTML 5 might result in various validation tools allowing constructs that JavaHelp can't deal with.

I'd love to get rid of JavaHelp, but until we do, we should probably keep the 4.01 doctype.

Reply:
My opinion is that the web access be given preference, therefor HTML 5.

My understanding is that HTML 5 is more about CSS and Javascript than about HTML itself.  I believe JavaHelp ignores CSS and scripts so the risk of using new HTML features is probably small.

As far as Window Help is concerned, any machine capable of running JMRI can run a browser.  The html files used by JavaHelp are included in the JMRI distribution.  This provides offline access to window help.

Reply:
Haven't seen big issues in JHelp in-line display of html(5) pages. We already serve a number of those under `help.en.html`
The way a JavaScript is tagged in html5 differs from previous definitions, so my intent here is to stop that being flagged as an html error. JHelp will put limits on actually using many of the newer html5 options.
(Note I left in place all PRAGMA redirect pages. They never display, and some of their content needs to be rewritten to comply).

Reply:
Can we agree on merging this set (a lot of JMRI help pages that show up in JHelp have already been migrated to html5 DOCTYPE. Discussion on JHelp alternative solutions to choose from may continue, as a separate topic/issue.

Reply:
I think proceeding is fine.  I did not run into any issues with JavaHelp with the LogixNG help pages which are all HTML 5.

Reply:
I have no opinion about this PR since I don't know much about html4 / 5 and its impact on JavaHelp.
Do you want this PR merged before or after 4.24 is released? If it's merged now, it will be in 4.24.

Reply:
Sorry, missed this general question inside the review discussion thread. Would be nice, but not essential (window may have closed by now)